src/hevm/src/EVM.hs

```diff
@@ -377,6 +381,7 @@ data Block = Block
   , _number      :: Word
   , _difficulty  :: Word
   , _gaslimit    :: Word
+  , _baseFee     :: Word
```

Comment:
I guess this needs to be a `Maybe` to account for pre-1559 blocks?

Reply:
we haven't bothered to make hevm backwards compatible, so I think this is fine as long as we signal that hevm is now on 1559

Reply:
I think it's gonna break `seth run-tx` on old blocks?

Reply:
Although yeah the prospect of worrying aobut backwards compatibility in hevm does not spark joy...